detections/endpoint/ssa___first_time_seen_dlls_lolbas.yml

```diff
@@ -0,0 +1,39 @@
+name: First time seen dlls lolbas
+id: 5f0b54ba-c884-4413-ba4d-5c187b6b17ed
+version: 1
+date: '2021-3-1'
+author: 'Joseph Zadeh, Splunk'
+type: streaming
+datamodel: []
+description: Identify dlls that have been observed for the first time
+  in windows active directory logs.
+search: '| from read_ssa_enriched_events()
+         | eval timestamp=ucast(map_get(input_event, "_time"), "long", null),
+           process=lower(ucast(map_get(input_event, "process"), "string", null)),
+            --for now ignore the "process" field since this contains both process_name plus command arguments
+           dest_user_id=ucast(map_get(input_event, "dest_user_id"), "string", null),
+           dest_device_id=ucast(map_get(input_event, "dest_device_id"), "string", null)
+         | where like(process, "%.dll")
```

Comment:
This is really not what we were looking for - it would match any command lines that have ".dll" in them.

Instead, this was the plan as per Jira TR-365 (https://jira.splunk.com/browse/TR-365) :
* Build a detection that identifies first times when a specific DLL is run.
* Consider looking for global/per-entire-customer-org first times vs. doing first times for a user or a device.
* Consider doing (another) simpler rule-based detection for DLLs that are not run from system folders.

I'll post some detection code in Jira, let's make datasets for it, and start from there.